README.md

```diff
@@ -1,34 +1,23 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-## Getting Started
-
-First, run the development server:
-
-```bash
-npm run dev
-# or
-yarn dev
-```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+## Local Installation
+Clone the source code:
+git clone https://github.com/projecttacoma/deqm-test-server-frontend.git
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+Install dependencies:
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+npm install
```

Comment:
Add info about creating a `.env.local` file with a reference to the `deqm-test-server`.